deterministic-bloom-wasm/src/lib.rs

```diff
@@ -27,29 +23,167 @@ pub fn set_panic_hook() {
     console_error_panic_hook::set_once();
 }
 
-#[wasm_bindgen]
-extern "C" {
-    // For alerting
-    pub(crate) fn alert(s: &str);
-    // For logging in the console.
-    #[wasm_bindgen(js_namespace = console)]
-    pub fn log(s: &str);
-}
-
 //------------------------------------------------------------------------------
 // Macros
 //------------------------------------------------------------------------------
 
-/// Return a representation of an object owned by JS.
-#[macro_export]
-macro_rules! value {
-    ($value:expr) => {
-        wasm_bindgen::JsValue::from($value)
+/// Generate a monomorphic [BloomFilter] newtypes.
+macro_rules! gen_bloom {
+    ($name: ident, $n: expr, $k: expr) => {
+        #[doc = concat!("A monomorphic wrapper for [BloomFilter]`<", stringify!($n), ", ", stringify!($k), ">`.")]
```

Comment:
For docs, I'd probably expand out what the $n and $k are more specifically, as it reads a bit weird when rendered w/ just the numbers within the angle brackets. 

<img width="997" alt="Screenshot 2023-01-09 at 10 57 48 AM" src="https://user-images.githubusercontent.com/202820/211351575-f637edcf-1268-4a96-ab8d-89003932aea6.png">
<img width="559" alt="Screenshot 2023-01-09 at 10 57 43 AM" src="https://user-images.githubusercontent.com/202820/211351581-c055fd78-faaa-40b1-bf2b-d263b3169114.png">